### ISSUE TITLE
Playlist sharing

### ISSUE THREAD

**Issue by CodeNameButtons** [vfsfitvnm/ViMusic#523]
### Feature description

So basically in the menu where you have the option to to delete or rename a playlist think it would be by cool to have a share button

### Why do you want this feature?

So I can share my playlists with friends

**Comment by vfsfitvnm** [comment]
The only possible thing to do is passing the video ids in a file, which is :neutral_face:

**Comment by Lolozweipunktnull** [comment]
I would like to have playlists exportable. I would even like the solution with the IDs file.

**Comment by arfanmrafeek** [comment]
@PassionateG1t , Yes but when you share it, If it has more than 100+ videos, After sharing, It's very hard to know the content of the video because it will export it like individual link for all videos in a Playlist

**Comment by vfsfitvnm** [comment]
I already have a working implementation by the way. I just want to make sure many people needs this before bloating the codebase

**Comment by selurvedu** [comment]
An idea for playlist saving and sharing:

Hyperpipe, an alternative YouTube Music frontend, allows you to share your playlists via a sharing code. See here: https://hyperpipe.surge.sh/library/

It also allows logging in with a Piped account if you have one.
There is no support for Invidious accounts though.

**Comment by Edotor** [comment]
Ok I see my request closed because there is this open! I hope that this feature will be ready as soon as possible. It will be a great thing

**Comment by lbrunkho** [comment]
I think it would be wise to avoid using another third party service. Using a linked YouTube account to host the playlist seems more likely but this will result in reduced privacy. My vote is for an m3u8 (playlist) export that would point to the full YouTube link within the playlist. This way other apps can possibly use this playlist file (VLC can play YouTube videos from just a link for example).

**Comment by Wave6677** [comment]
> I already have a working implementation by the way. I just want to make sure many people needs this before bloating the codebase

I can tell you I'd greatly appreciate it. I have a lot of friends who ask me for playlist recommendations but I can't share anything because I use ViMusic.

**Comment by bebo2855** [comment]
Yea we need this feature

**Comment by gbucar** [comment]
I would surely use it and if it is already implemented, why not add it to the code?

**Comment by Terramoto** [comment]
Basic export/import would do for me, Nuclear has this feature but it uses a proxy so i don't think it uses the actual IDs, instead it uses internal ones.
